gatsby/src/docs/accounts/sso/azure-sso.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Azure Active Directory SSO'
+sidebar_order: 1
+---
+
+## Installation
+
+1. Log in to the [Azure portal](https://portal.azure.com/).
+
+2. (If you don't require a permission group for Sentry, skip to Step 3.) Under Manage Azure Active Directory (the shield picture), click "View", which takes you to the Organization Overview page. Now, proceed to Step 3.
+
+3. On the left hand navigation, click "Groups". Create a new group, add an owner, and members to the group. Set "Group type" to Office 365. For more details about group creation, see the [Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+4. Return to the Overview page. In the search bar, search for "Enterprise Applications" and navigate to "Enterprise Applications." Then, select "Non-gallery application" and create a non-gallery application for Sentry. Once created, you'll be directed to "Sentry - Overview.
```

Comment:
@vudngo I rewrote this step. Let me know if we can improve it. 😄

Reply:
this is perfect.